source/SIL.AppBuilder.Portal.Frontend/src/data/containers/with-role.tsx

```diff
@@ -205,7 +205,7 @@ export function withRole<TWrappedProps extends {}>(
   };
 }
 
-function canDoEverything(dataStore: Store, currentUser: UserResource) {
+export function canDoEverything(dataStore: Store, currentUser: UserResource) {
```

Comment:
This feels somewhat like a wrapper of `isSuperAdmin` Looks like the only difference is you are passing currentUser vs the userRoles. Reads well though `const isSuperAdmin = currentUser && canDoEverything(dataStore, currentUser);`

So no action needed. Was thinking maybe changing name to `isUserSuperAdmin` or something...but it's synonymous.

Reply:
good call tho